include/mega/filesystem.h

```diff
@@ -112,7 +112,7 @@ struct MEGA_API FileAccess
     bool fread(string *, unsigned, unsigned, m_off_t);
 
     // absolute position read to byte buffer
-    bool frawread(byte *, unsigned, m_off_t);
+    virtual bool frawread(byte *, unsigned, m_off_t);
```

Comment:
I don't think this should become virtual, that's affecting/slowing production code - the tests that want to use this can override the four customization functions that are already virtual per the design: sysread/sysstat/sysopen/sysclose - just like a customization for a particular platform, but that platform is virtual. thanks!

Reply:
Ok. That should be doable. Will fix